lighthouse-viewer/app/styles/viewer.css

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --lh-background-color: rgb(32,50,70);
+}
+
+.drop_zone {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  visibility: hidden;
+}
+.drop_zone.dropping {
+  visibility: visible;
+  font-size: var(--heading-font-size);
+  background: rgba(255,255,255,0.8);
+  color: var(--unknown-color);
+}
+.drop_zone.dropping::after {
+  content: 'Drop report here';
+  border: 2px dashed currentColor;
+  border-radius: 5px;
+  padding: 25px;
+  width: 33vw;
+  height: 20vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: white;
+}
+.viewer-placeholder {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+  background-color: var(--lh-background-color);
+}
+.viewer-placeholder-inner {
+  display: flex;
+  align-items: center;
+  border-radius: 5px;
+  padding: 40px 32px;
+  border: 2px dashed rgba(255,255,255,0.2);
+  cursor: pointer;
+}
+.viewer-placeholder-inner.dropping {
+  border-color: currentColor;
+}
+.viewer-placeholder h1 {
```

Comment:
We want to keep specificity low (not that it matters to much on a small project) so why do the nesting of viewer-placeholder h1 which we could change it into

```
.viewer-placeholder__title {
font-weight: 300;
}
```
or
```
.u-font-weight--light {
font-weight: 300;
}
```